src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java

```diff
@@ -64,6 +79,56 @@
     }
 
     @Override
+    @SuppressWarnings({"unchecked", "rawtypes"})//FIXME generics here blow up
+    public Map<String, Object> combineRawData(final List<Allele> vcAlleles, final List<? extends ReducibleAnnotationData> annotationList) {
```

Comment:
The unchecked/rawtypes has to do with the untyped annotation classes

Reply:
I used to have RMSMappingQuality extending RMSAnnotation, though admittedly it was the only class that extended RMSAnnotation.  I did that because someday there might be other data we want to take the root-mean-square of, but that probably pretty unlikely.